Scripts/DMX/DmxTalker.cs

```diff
@@ -0,0 +1,327 @@
+using System;
+using System.Collections.Concurrent;
+using System.Timers;
+using Avalonia;
+using DynamicData.Tests;
+using Haukcode.sACN;
+using YALCY.ViewModels;
+
+namespace YALCY.Scripts.DMX
```

Comment:
In other scripts, brace-less namespaces are used, however here, a namespace declaration with braces are used! Stick to a consistent style.